chamber/storages/boto3.py

```diff
@@ -0,0 +1,28 @@
+from django.core.files.base import ContentFile
+
+from storages.backends.s3boto3 import S3Boto3Storage
+
+
+__all__ = (
+    'S3Storage',
+    'force_bytes_content',
+)
+
+
+def force_bytes_content(content):
+    _content = content.read()
+    casted = False
+    if not isinstance(_content, bytes):
+        _content = bytes(
+            _content,
+            'utf-8' if not hasattr(content, 'encoding') or content.encoding is None else content.encoding,
+        )
+        casted = True
+    return ContentFile(_content), casted
```

Comment:
to je v pohodě že tu vracíš dvojičku a níže s tím nepočítáš?

Reply:
Není, uteklo mi to.